docs/contributors/workflow.rst

```diff
@@ -1,25 +1,8 @@
-Development guidelines
-======================
-
-Philosophy
-----------
-
-The mailserver is designed as a whole, some images are therefore not best
-suited for reuse outside this project. All images should however follow
-Docker best practices and be as generic as possible :
-
- - even if not suited for reuse, they should be simple enough to
-   fit as base images for other projects,
- - interesting settings should be available as environment variables
- - base images should be well-trusted (officiel Alpine or Debian for instance).
-
-.. _git_workflow:
```

Comment:
https://github.com/Mailu/Mailu/blob/master/docs/contributors/environment.rst#git links to this deleted reference.